status/status_test.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("Status Adapter", func() {
 		pipelineRun = &tektonv1beta1.PipelineRun{
 			ObjectMeta: metav1.ObjectMeta{
 				Labels: map[string]string{
-					"pipelinesascode.tekton.dev/git-provider": "github",
+					"pac.test.appstudio.openshift.io/git-provider": "github",
```

Comment:
Not to seem repetitive, but just imagine how much smaller the diff would have been if these string literals were referencing a string constant declared somewhere central. Something to think about.

Reply:
A constant has been created but I didn't use it in test-related code. This is intentional to make it difficult to accidentally change the prefix without knowingly correcting the expected values in the tests.